keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -280,13 +279,21 @@ _This section must be completed when targeting beta graduation to a release._
   an object specifying the range and validating if the traffic is allowed within 
   the specified range
 
+* **How can someone using this feature know that it is working for their instance?
+  - [x] Other
+   - Details:
+      The API Field must be present when a NetworkPolicy is created with that field.
+      The feature working correctly depends on the CNI implementation, so it can be assessed
+      by the operator verifying if the data path contains the correct network filtering rules 
+      and also testing if communications works as desired.
+
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
   Operators would need to monitor the traffic of the Pods to verify if a 
   specified port range is applied and allowed in their workloads
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
- N/A
+ - per-day percentage of API calls finishing with 5XX errors <= 1% is a reasonable SLO
```

Comment:
Can't comment there - but CNI is a dependency for this feature, right? (line 305)

Reply:
Yes, will change in the dependencies line